autobahn/wamp/interfaces.py

```diff
@@ -262,7 +262,10 @@ class ISession(object):
     @abc.abstractmethod
     def onConnect(self):
         """
-        Callback fired when the transport this session will run over has been established.
+        Callback fired when the transport this session will run over has
+        been established.
+
```

Comment:
*if* we decide to do that (which we need to discuss first since it is API changing), *then* we should do that in a different commit. In any case, looking at the impacts takes some thinking and time.